### PR TITLE
feat: customize graphql for resources frontend

### PR DESCRIPTION
**Pull request by gjuro87**

### CODE REVIEW

**Comment by gjuro87** [comment]
Everything is done. There is one issue that i solved in another way. Issue when calling dependency graphql query and requesting id. Result is error: "'Dependency' object has no attribute 'pk'". I couldn't find the cause. I have solve it another way, not requesting id, but needs to be solved. If you could take a look

**Comment by gjuro87** [comment]
Bug fixed, thx for help

File: frontend/packages/zoo-api/src/queries.ts [hunk 1]
```diff
@@ -394,54 +452,24 @@ export interface ActiveIncident extends Node {
   color: string;
 }
 
-export interface Resource extends Node {
+export interface Dependancy extends Node {
```
Comment (blfpd) [inline frontend/packages/zoo-api/src/queries.ts]: Could you replace Depend**a**ncy by Depend**e**ncy in this MR?

**Review by blfpd — CHANGES_REQUESTED** [review]
Annoying typo. Otherwise good MR! :blush:

**Comment by blfpd** [comment]
For the Pytest check, run `pytest test/api/query/test_node.py --snapshot-update` and review https://github.com/kiwicom/the-zoo/blob/hangar/test/api/query/snapshots/snap_test_node.py

File: frontend/packages/zoo-api/src/queries.ts (hunk 1 above)
Comment (gjuro87) [reply frontend/packages/zoo-api/src/queries.ts]: Sorry, I am little bit dyslexic;) Fixed

**Pull request merged**